<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StageManager</class>
 <widget class="QMainWindow" name="StageManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StageManager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="StageManager::ui::ProcessTabWidget" name="stageTabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>781</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addStageTabButton">
       <property name="text">
        <string>AddTab</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="configurationButton">
       <property name="text">
        <string>Configuration</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConfiguration">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StageManager::ui::ProcessTabWidget</class>
   <extends>QTabWidget</extends>
   <header location="global">../ui/ProcessTabWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
